webapi/Extensions/SemanticKernelExtensions.cs

```diff
@@ -33,7 +33,7 @@ internal static class SemanticKernelExtensions
     /// <summary>
     /// Delegate to register plugins with a Semantic Kernel
```

Comment:
This says "register plugins", while line 36 change says "registerFunctions..."  Can this please be clarified?  Seeing the terms plugin and function used interchangeably elsewhere too.

Reply:
Plugin has one or more functions.  The plug in registers functions that are called as planner steps.  Just trying to align with how the term is being used in semantic-kernel.